code/xcvm/lib/core/src/prelude.rs

```diff
@@ -5,8 +5,8 @@ pub use alloc::{
 	vec,
 	vec::Vec,
 };
-pub use core::str::FromStr;
-pub use cosmwasm_std::{Addr, Binary, Coin, Uint128};
+pub use core::{fmt::Display, str::FromStr};
+pub use cosmwasm_std::{Addr, Binary as Base64Binary, Coin, HexBinary, Uint128};
```

Comment:
No.  Having prelude is bad enough.  Having prelude where a type is renamed is definitely a NAK.

Reply:
what nak means?

Reply:
it is type alias. so we should not use type aliases?

Reply:
if we use any existing type alias in prelude than we should not use that too?

why?

Reply:
NAK = Negative Acknowledgement.

We should not be using preludes or wildcard imports at all since they make reading code harder.  But since you insist on using preludes, we definitely should not invent new names in a prelude.

Reply:
so we should not use type aliases in prelude?

Reply:
Let align with others on the case